internal/pkg/earpointer/analysis.go

```diff
@@ -315,7 +345,134 @@ func (vis *visitor) visitSelect(s *ssa.Select) {
 }
 
 func (vis *visitor) visitCall(call *ssa.CallCommon, instr ssa.Instruction) {
-	// TODO: to be added.
+	if builtin, ok := call.Value.(*ssa.Builtin); ok { // special handling of builtin functions
+		vis.processBuiltinCall(builtin, instr)
+		return
+	}
+
+	// Collect unification constraints
+	paramCstrs, retCstrs := vis.collectCalleeConstraints(call, call.StaticCallee(), instr)
+	// Unify caller arguments and callee parameters with suitable context.
+	for arg, params := range paramCstrs {
+		for _, param := range params {
+			// The copy-by-value semantics is implemented by the SSA call. For
+			// example:
+			//   type S struct { ... }
+			//   var v S
+			//   g(v)
+			//   func g(a S) { ... }
+			// SSA creates a copy of v and passes it to g. Hence we directly unify
+			// this copy and g's argument "a".
+			if mayShareObject(arg) {
+				vis.unifyLocals(arg, param)
+			}
+		}
+	}
+	// Handle return value unification.
+	for callerRet, rets := range retCstrs {
+		for _, calleeRet := range rets {
+			if len(calleeRet) == 1 { // non-tuple case
+				if mayShareObject(calleeRet[0]) {
+					vis.unifyLocals(callerRet, calleeRet[0])
+				}
+			} else { // tuple case
+				// For call "return_r = f(...)", if the callee returns a tuple <r0, r1, ...>,
+				// then the i^{th} element in the tuple is connected to the caller's return_r
+				// by making "return_r[i] = r{i}".
+				for k := 0; k < len(calleeRet); k++ {
+					retV := calleeRet[k]
+					if !mayShareObject(retV) {
+						continue
+					}
+					// Implement "return_r[i] = r{i}".
+					field := Field{Name: strconv.Itoa(k)}
+					vis.processHeapAccess(retV, callerRet, field)
+				}
+			}
+		}
+	}
+}
+
+// Process calls to builtin functions; return true if the callee is a builtin
+// function.
+// Builtin functions: https://golang.org/pkg/builtin/
+func (vis *visitor) processBuiltinCall(builtin *ssa.Builtin, instr ssa.Instruction) {
```

Comment:
You seem to have a well-defined sense of `visit` vs `process`, but that distinction is a bit opaque to me.  I might've guessed this should be named `visitBuiltinCall`, since it seems to be at that higher visitor level.

If there is a consistent theme in the verbiage for these methods, I'd find a top-level summary comment near the `package` declaration useful.